Helm/opensearch-dashboards/templates/configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.config }}
```

Comment:
Should this match the technique used in the other templates like this?
```
{{- if .Values.config -}}
...
{{- end }}
```

Reply:
I guess it can remain like this as well.